src/policy/fees.cpp

```diff
@@ -365,7 +367,15 @@ bool CBlockPolicyEstimator::processBlockTx(unsigned int nBlockHeight, const CTxM
     // Feerates are stored and reported as BTC-per-kb:
     CFeeRate feeRate(entry->GetFee(), entry->GetTxSize());
 
-    feeStats.Record(blocksToConfirm, (double)feeRate.GetFeePerK());
+    // Dogecoin: Pre-1.14.4 transaction sizes were rounded, which led to effective fee rates higher
+    // than 1 DOGE. These are considered ridiculous currently, so cap them at 1 DOGE until the fees
+    // stabilise.
+    if (feeRate.GetFeePerK() > HIGH_TX_FEE_PER_KB) {
```

Comment:
why not use `MAX_FEERATE` from `policy/fees.h`? Also saves an include of `validation.h`

Reply:
Could do - the answer is because `MAX_FEERATE` was higher than `HIGH_TX_FEE_PER_KB`, but in hindsight having capped the values we use, might as well cap the bucket there too.

Reply:
Done, re-testing the results now